binders/kafka-binder/spring-cloud-stream-binder-kafka/src/main/java/org/springframework/cloud/stream/binder/kafka/KafkaBinderMetrics.java

```diff
@@ -64,7 +65,7 @@
  * @author Lars Bilger
  */
```

Comment:
Please add your name as an author. Also, make sure that any Java classes modified have the updated copyright years. Thanks!